group_vars/talos_cluster/vars.yml

```diff
@@ -56,6 +56,7 @@ ldap_domains:
     user_name: uid
     user_ssh_public_key: sshPublicKey
     user_member_of: groupMembership
+    user_expiration_date: loginExpirationTime
```

Comment:
Are this all the clusters that will need this new ldap file? I am missing quite some number of clusters in it (BB, CF, FH).

Reply:
For now: yes. Let's deal with changes required due to updates from recent maintenance first. The others can follow later.

Reply:
It should not fail; I added defaults for this to [roles/sssd/templates/readonly-ldapsearch-credentials.bash](https://github.com/rug-cit-hpc/league-of-robots/pull/684/files#diff-95ba3d09d16e8acf26e2dad49d11f642cb3abf30405f725bf50341f0b611efb1)

Reply:
Ok. I thought that the `ldapsearch` will later fail, since it will not have this credentials available. But if this is not the case, then perfect. Let's merge it.